README.md

```diff
@@ -8,7 +8,8 @@ Don't play for too long! We need you to write the code!
 
 Okay, what do we have?
 1) HTML and CSS are already written. You can use it, or implement your own design if you want.
-2) Reference.
+2) Base `Game` class structure is already written too. Extend it with your own methods.
```

Comment:
please also expain that tests check the code using ... methods
Otherwise it may not be clear that they should keep them